.gitignore

```diff
@@ -27,3 +27,4 @@ setup.log
 
 # Local OPAM switch
 _opam/
```

Comment:
```suggestion
_opam/
_build/
```

Reply:
the build repertory is already in the gitignore